picky-server/CHANGELOG.md

```diff
@@ -1,5 +1,27 @@
 # Changelog
 
+## [4.5.0] 2020-03-18
```

Comment:
I would suggest to keep the date empty for now or write 2020-xx-yy.  We will release it probably later, when it will be needed